src/redux/cryptoreducer.jsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable */
```

Comment:
- [ ] I noticed that you disable a lot **Eslint** . Please remove the **eslint disable** statement and try to fix all the errors you are getting. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1: